service/service.ml

```diff
@@ -54,11 +54,14 @@ module Make (Opam_repo : Opam_repository_intf.S) = struct
       worker#terminate;
       worker#close >|= function
       | Unix.WEXITED code ->
-          Fmt.epr "Worker %d finished@. Exited with code %d" pid code
+          Fmt.epr "Worker %d finished. Exited with code %a@." pid
+            Fmt.Dump.signal code
```

Comment:
Maybe it's unnecessary to log when the process exits successfully (exit code 0)?

Reply:
Also true.